labelbox_dev/task.py

```diff
@@ -221,3 +223,77 @@ def errors(self) -> Union[Dict[str, Any], None]:
             errors['failed_data_rows'] = task_result['failedDataRows']
 
         return errors if errors else None
+
+
+class UpdateDataRowsTask(BulkTask):
+    """ Asynchronous task representing bulk data_rows update job
+    """
+
+    task_type = TaskType.UPDATE_DATA_ROWS
+
+    def __init__(self, json):
+        super().__init__(json)
+
+    @lru_cache()
```

Comment:
why are we using result cache in an update call? is this for call de-duping?

Reply:
This is going to return the resulting set of data rows updated. If customer fetches this result set multiple times, we don't want to fetch it from GCS every time, hence the usage of @lru_cache